serving-api/src/main/scala/com/stratio/sparta/serving/api/service/http/PolicyHttpService.scala

```diff
@@ -129,8 +129,9 @@ trait PolicyHttpService extends BaseHttpService with SpartaSerializer {
     path(HttpConstant.PolicyPath / "fragment" / Segment / Segment) { (fragmentType, id) =>
       get {
         complete {
-          val future = supervisor ? new FindByFragment(fragmentType, id)
-          Await.result(future, timeout.duration) match {
+          for{
+            response <- supervisor ? FindByFragment(fragmentType, id)
+          }yield response match {
```

Comment:
correct indentation and spaces here:
for{
...
}yield

to

for {
...
} yield